doc/sphinx/practical-tools/coqide.rst

```diff
@@ -91,9 +91,12 @@ from the toolbar and from the keyboard.  These include:
 - Run to cursor (`Ctrl-Right`) to run commands up to the cursor
 - Run to end (`Ctrl-End`) to run commands to the end of the buffer
 - Reset Coq (`Ctrl-Home`) to restart the Coq process
-- Interrupt to stop processing commands after the current command completes
+- Interrupt to stop processing commands after the current command completes.
+  (Note: on Windows but not on WSL, Interrupt doesn't work if you start CoqIDE
```

Comment:
I would tend to recommend using a `.. warning::` in this case, but let's merge as is.